assets/models/system/CodeLlama-13b-Python-hf/spec.yaml

```diff
@@ -0,0 +1,19 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: CodeLlama-13b-Python-hf
+path: ./
+properties:
+  SHA: 488587ade99c84ea4325e6d620d307372728cc97
+  languages: EN
+tags:
+  Featured: ''
+  Preview: ''
+  inference_compute_allow_list:
+  - Standard_NC24s_v3
+  - Standard_NC24rs_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_v4
```

Comment:
Could you please check the SKU names with @srsaggam ?

Reply:
Sure

Reply:
Done @skasturi , changed